usr/share/rear/setup/default/010_pre_recovery_script.sh

```diff
@@ -1,4 +1,6 @@
 if test "$PRE_RECOVERY_SCRIPT" ; then
-	Log "Running PRE_RECOVERY_SCRIPT '${PRE_RECOVERY_SCRIPT[@]}'"
-	eval "${PRE_RECOVERY_SCRIPT[@]}"
+    Log "Running PRE_RECOVERY_SCRIPT '${PRE_RECOVERY_SCRIPT[@]}'"
```

Comment:
One should use `${PRE_RECOVERY_SCRIPT[*]}` inside a larger quoted array, and leave the `${PRE_RECOVERY_SCRIPT[@]}` only when directly surrounded by quotes, because it splits the containing string into multiple ones: `PRE_RECOVERY_SCRIPT=(foo bar)` results in `Log "Running PRE_RECOVERY_SCRIPT foo" "bar"` instead of `Log "Running PRE_RECOVERY_SCRIPT foo bar"`. In the case of `Log` it fortunately does not matter, but it is generally not what's intended.

Reply:
New code pushed, please re-review